packages/client-core/src/admin/common/ConfirmModel.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import Dialog from '@mui/material/Dialog'
+import DialogActions from '@mui/material/DialogActions'
+import DialogTitle from '@mui/material/DialogTitle'
+import Button from '@mui/material/Button'
+import { useStyles } from '../styles/ui'
+
+interface Props {
+  popConfirmOpen: boolean
+  handleCloseModel: () => void
+  submit: () => void
+  name: string
+  label: string
+}
+
+const ConfirmModel = (props: Props) => {
+  const classes = useStyles()
+  const { popConfirmOpen, handleCloseModel, submit, name, label } = props
+  return (
+    <Dialog
+      open={popConfirmOpen}
+      onClose={handleCloseModel}
+      aria-labelledby="alert-dialog-title"
+      aria-describedby="alert-dialog-description"
+      classes={{ paper: classes.paperDialog }}
+    >
+      <DialogTitle id="alert-dialog-title">
+        Do you want to delete {label} <b>{name}</b>
```

Comment:
Please add a `?` at the end of the title.

Reply:
Do you mean like thins ``name?`` ?

I am not able to get reason, I should do that

Reply:
since it's a question there should be a question mark at the end of the title